CRADLE/CorrectBiasStored/calculateOneBP.pyx

```diff
@@ -1,3 +1,5 @@
+# cython: language_level=3
```

Comment:
Would python 2.* have a problem?

Reply:
I don't know but at this point there is no reason to support Python 2. Since this is a new project I think we should aim for Python 3.7 to be the minimum supported version. Pythons 2.7 - 3.5 have all reached "end of life" and are neither supported by nor getting new releases from the Python Software Foundation. Python 3.6 reaches the "end of life" stage at the end of this year, so Python 3.7 seems like a good target for longer-term support. Additionally PyPy3, a common alternative Python implementation, supports Python 3.7(.9).